pkg/infoblox/ibmock/client.go

```diff
@@ -41,33 +43,33 @@ func (m *MockClient) EXPECT() *MockClientMockRecorder {
 }
 
 // CheckNetworkExists mocks base method.
-func (m *MockClient) CheckNetworkExists(arg0 string, arg1 netip.Prefix) (bool, error) {
+func (m *MockClient) CheckNetworkExists(view string, subnet netip.Prefix) (bool, error) {
```

Comment:
did you manually modify this, or does gomock's generator now use better names for parameters?

Reply:
nice